.changeset/cool-kiwis-shop.md

```diff
@@ -0,0 +1,12 @@
+---
+"@comet/admin-theme": major
+---
+
+Rework theme of MUI's `Chip`
+
+The main changes include:
+
+-   Increase border radius to 12px (from 4px)
+-   Medium height of 24px (previously 32px)
+-   Small height of 20px (previously 24px)
+-   Default delete icon `Clear` from `@comet/admin-icons`
```

Comment:
I don't think all this detail is really necessary for the changelog, just mentioning the Chip's default styling was changed to match the updated Comet CI should be enough imo.

Reply:
```suggestion
Rework theme of MUI's `Chip` to match the updated Comet CI
```